src/Resolvers/Microsoft.DotNet.NativeWrapper/EnvironmentProvider.cs

```diff
@@ -33,12 +35,12 @@ private IEnumerable<string> SearchPaths
             {
                 if (_searchPaths == null)
                 {
-                    var searchPaths = new List<string>();
-
-                    searchPaths.AddRange(
+                    var searchPaths =
```

Comment:
nit:
Maybe replace this whole block with:
_searchPaths ??=
...

?

Reply:
Done!